### PR TITLE
Visual improvements for the list of job in UI

### PR DESCRIPTION
**Pull request by ubuntolog**
Now we use React Bootstrap Table component. Jobs table is sortable, rows are expandable and display much more information (e.g. job duration time). Multiple job removal is possible. Closes https://github.com/EUDAT-GEF/GEF/issues/122

### CODE REVIEW

File: webui/src/components/Jobs.jsx [hunk 1]
```diff
@@ -1,60 +1,252 @@
...
+        if (hours   < 10) {hours   = "0"+hours;}
+        if (minutes < 10) {minutes = "0"+minutes;}
+        if (seconds < 10) {seconds = "0"+seconds;}
+        return hours+':'+minutes+':'+seconds;
+    }
+
+
+    hasJobsRunning() {
+        var runningJobfound = false;
+        this.props.jobs.map((job) => {
```
Comment (emanueldima) [inline webui/src/components/Jobs.jsx]: this should also work: 
````
hasJobsRunning() { 
    const firstRunningJob = this.props.jobs.find(job => job.State.Code < 0); 
    console.log(firstRunningJob.State);
    return firstRunningJob;
````

File: webui/src/components/Jobs.jsx [hunk 2]
```diff
@@ -1,60 +1,252 @@
...
+
+            if (job.State.Code == -1) {
+                console.log(job.State);
```
Comment (emanueldima) [inline webui/src/components/Jobs.jsx]: necessary?

File: webui/src/components/Jobs.jsx [hunk 3]
```diff
@@ -1,60 +1,252 @@
...
+        failedJobs = 0;
+
         if (this.props.jobs) {
             return (
                 <div>
                     <h3>Browse Jobs</h3>
-                    <h4>All jobs</h4>
-                    <Header/>
-                    { this.props.jobs.map((job) => {
+                    {this.props.jobs.map((job) => {
```
Comment (emanueldima) [inline webui/src/components/Jobs.jsx]: extract the following code into a function, it will be cleaner

File: webui/src/components/Jobs.jsx (hunk 1 above)
Comment (ubuntolog) [inline webui/src/components/Jobs.jsx]: This one does not work. firstRunningJob is always undefined

File: webui/src/components/Jobs.jsx (hunk 2 above)
Comment (ubuntolog) [inline webui/src/components/Jobs.jsx]: No, it is not

File: gefserver/server/api.go [hunk 4]
```diff
@@ -412,6 +413,7 @@ func (s *Server) removeJobHandler(w http.ResponseWriter, r *http.Request) {
 	}
 
 	err = s.db.RemoveJob(user.ID, jobID)
+	fmt.Println()
```
Comment (emanueldima) [inline gefserver/server/api.go]: unnecessary

**Pull request merged**